### ISSUE TITLE
Lcfirst should be applied to $tax_object->graphql_plural_name when creating custom post types

### ISSUE THREAD

**Issue by isvaljek** [wp-graphql/wp-graphql#892]
To reproduce on v0.3.4, on any platform/playground:
`{
  "input": {  
    "clientMutationId": "CreateExcursion",
    "title": "Excursion to Springfield",
    "destinations": {
    	"append": true,
      "nodes": [{"id": "b2RyZWRpc3RlOjYy"}]
    }    
  }
}`

I expected to see the new CPT created, and the custom taxonomy term designated by the id field appended to it.
The post got created, but no terms were appended.

This happens because I registered the custom taxonomy for graphql by Uppercase in functions.php, and the lines 262 and 264 of src/Data/PostObjectMutation.php expect lowercase:

```
/**
* If there is input for the taxonomy, process it
*/
if ( ! empty( $tax_object->graphql_plural_name ) && ! empty( $input[ $tax_object->graphql_plural_name ] ) ) {

$term_input = $input[ $tax_object->graphql_plural_name ];
```

**Comment by jasonbahl** [comment]
@isvaljek thanks for reporting. I'll see what I can find out.

**Comment by CodeProKid** [comment]
Might be related to #960 ?

**Comment by ddresch** [comment]
Thanks @isvaljek for this finding. I can confirm in version 0.4.0 the `lcfirst` is needed.

```
if ( ! empty( $tax_object->graphql_plural_name ) && ! empty( $input[ lcfirst($tax_object->graphql_plural_name) ] ) ) {
	$term_input = $input[ lcfirst($tax_object->graphql_plural_name) ];
```

**Comment by jasonbahl** [comment]
Fixed in #1025
